spec/miq_ae_uri_spec.rb

```diff
@@ -16,9 +16,9 @@
   end
 
   it "escape non-ASCII Numeric characters" do
-    hash = {"test://dev?lab$~" => "OU=serverbuildingtest, OU=dev2"}
+    hash = {"test://dev?lab$" => "OU=serverbuildingtest, OU=dev2"}
     query = described_class.hash2query(hash)
-    expect(query).to eq("test%3A%2F%2Fdev%3Flab%24%7E=OU%3Dserverbuildingtest%2C%20OU%3Ddev2")
+    expect(query).to eq("test%3A%2F%2Fdev%3Flab%24=OU%3Dserverbuildingtest%2C%20OU%3Ddev2")
```

Comment:
Should the `%7E` be replaced with a `~` instead of removing it from the test string?

Reply:
Then the test would need to be conditional based on the version of ruby since it is processed differently in 2.4 and 2.5 as seen here: https://travis-ci.org/ManageIQ/manageiq-automation_engine/builds/496782188

Reply:
@bdunne What do you think?  I'm ok with dropping it as it seems like we would just be testing ruby at that point.

Reply:
:+1: I'm okay with dropping it.  I didn't know if it was an important use case